### PR TITLE
Functional tests - Add some tests in module manager page

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Add some tests in module manager page
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Nightly => https://github.com/nesrineabdmouleh/testing_pr/actions/runs/4795060772
| Fixed ticket?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

File: tests/UI/pages/BO/modules/moduleManager/index.ts [hunk 1]
```diff
@@ -86,68 +161,144 @@ class ModuleManager extends BOBasePage {
...
+
+    return this.elementVisible(page, this.moduleBlock(module.tag), 10000);
+  }
+
+  /**
+   * Is bulk actions button disabled
+   * @param page {Page} Browser tab
+   * @return {Promise<boolean>}
+   */
+  isBulkActionsButtonDisabled(page: Page): Promise<boolean> {
```
Comment (Progi1984) [inline tests/UI/pages/BO/modules/moduleManager/index.ts]:
```suggestion
  async isBulkActionsButtonDisabled(page: Page): Promise<boolean> {
```

File: tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/07_bulkActions.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+// Import pages
+import dashboardPage from '@pages/BO/dashboard';
+import moduleManagerPage from '@pages/BO/modules/moduleManager';
+
+import {expect} from 'chai';
+import {BrowserContext, Page} from 'playwright';
+
+// Import data
+import Modules from '@data/demo/modules';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/07_bulkActions.ts]:
```suggestion
// Import data
import Modules from '@data/demo/modules';

import {expect} from 'chai';
import {BrowserContext, Page} from 'playwright';
```

File: tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/06_installUninstallModule.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+// Import pages
+import dashboardPage from '@pages/BO/dashboard';
+import moduleManagerPage from '@pages/BO/modules/moduleManager';
+
+import {expect} from 'chai';
+import {BrowserContext, Page} from 'playwright';
+
+// Import data
+import Modules from '@data/demo/modules';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/06_installUninstallModule.ts]:
```suggestion

// Import data
import Modules from '@data/demo/modules';

import {expect} from 'chai';
import {BrowserContext, Page} from 'playwright';
```

**Review by Progi1984 — COMMENTED** [review]
Awesome PR. 

Good job :muscle: 

Some feedbacks but they are minimal.

File: tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/06_installUninstallModule.ts (hunk 3 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/06_installUninstallModule.ts]: Done thx :)

**Comment by Progi1984** [comment]
@nesrineabdmouleh ESLint is not ok !

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Pull request merged**